ccd-definition/FixedLists/CareSupervision/ManageDocumentTypeListLA.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ManageDocumentTypeListLA",
+    "ListElementCode": "FURTHER_EVIDENCE_DOCUMENTS",
+    "ListElement": "Further evidence documents",
+    "DisplayOrder": 1
```

Comment:
need to double check radio button order with UX (design was specified but want to double check)